prisma/migrations/20211222181246_add_sc_address/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "EventType" ADD COLUMN     "scAddress" TEXT;
```

Comment:
What does `scAddress` means? It's not clear from the name itself. Should we try to be as legible as possible?

Reply:
Should we adopt a similar `metadata` approach as users for event types? I think these kind of things are a perfect fit for it since they're not part of core functionality. Stripe also allows to save metadata in almost all their Models.

Reply:
It is better to have the `metadata` column as @zomars  says

Reply:
> Should we adopt a similar `metadata` approach as users for event types? I think these kind of things are a perfect fit for it since they're not part of core functionality. Stripe also allows to save metadata in almost all their Models.

@PeerRich, who should handle this?

Reply:
I agree, metadata would be a better way. @edanfesi can you make that change?

Reply:
yes, I am doing this change